test/functional/sqlsrv/TC34_PrepAndExec.phpt

```diff
@@ -6,18 +6,25 @@ Validates that a prepared statement can be successfully executed more than once.
 --ENV--
 PHPT_EXEC=true
 --SKIPIF--
-<?php require('skipif_versions_old.inc'); ?>
+<?php 
+// locale must be set before 1st connection
+if (!isWindows()) {
+    setlocale(LC_ALL, "en_US.ISO-8859-1");
+}
+require('skipif_versions_old.inc');
```

Comment:
We don't want to add this skipif because we do want to run this the UTF8 test for older versions. That was the whole point of isLocaleSupported, it does the checking there so we can do the skip inside each test. That was actually your idea, remember :)

Reply:
No worries. This skipif will only skip the test if AE is enabled. Otherwise, the test will still run